dotnet/ServiceBusDotNetTests.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Please add `dotnet` in .npmignore to ensure we do not publish this by accident.

Reply:
Done.